data/events/scripts/monster.lua

```diff
@@ -0,0 +1,37 @@
+function Monster:onDropLoot(corpse)
+	if (configManager.getNumber(configKeys.RATE_LOOT) == 0) then
+		corpse:decay()
+		return
+	end
+	
+	local player = Player(corpse:getCorpseOwner())
+	local mType = self:getType()
+	if not player or player:getStamina() > 840 then
+		local monsterLoot = mType:getLoot()
+		for i = 1, #monsterLoot do
+			local item = corpse:createLootItem(monsterLoot[i])
+			if not item then
+				print('[Warning] DropLoot:', 'Could not add loot item to corpse.')
+			end
+		end
+
+		if player then 
```

Comment:
if not player (line 9) and now if player?

Reply:
not player or player:getStamina() > 840
if there is no player it should also create the loot.